lib/api/decorators/single.rb

```diff
@@ -65,7 +65,14 @@ def initialize(model, current_user:, embed_links: false)
                render_nil: false,
                writable: false
 
+      # Defines associations that shall be eager_load'ed when rendering entities represented by this decorator.
+      # This is a hint for renderers of this representer.
       class_attribute :to_eager_load
+
+      # Defines associations that shall be preload'ed when rendering entities represented by this decorator.
+      # This is a hint for renderers of this representer.
```

Comment:
"Defines" here makes it look like it does a lot, while it seems to be just a list.
```suggestion
      # List of associations that shall be preloaded when rendering entities represented by this decorator.
      # This is a hint for renderers of this representer.
```

Reply:
Agreeing to "List of". I'd still keep the "preload'ed" to be consistent with the "eager_load'ed" above.

Also open to format this differently, the intention though is, that "preload" here refers to the name of an active record method.